src/quo2/components/list_items/channel.cljs

```diff
@@ -59,6 +62,5 @@
                 (not muted?)
                 (not (pos? (int mentions-count)))
                 unread-messages?)
-       [quo2.icons/icon :i/channel-notification
-        {:size     20
-         :no-color true}])]]])
+       [rn/view
```

Comment:
maybe it's better to make this a reusable component that can also be used in other components? 🤔 that is if we have this dot on other components?

Reply:
Yes, we have it in `community_list_view.cljs`. I didn't create a separate component, but moved this style on the level of components and reused it in both places, wdyt?

Reply:
ah okay, it looks good actually. The only thing I would recommend is to put it in a named folder otherwise the file "src/quo2/components/style.cljs" will just become a plethora of random styles.

How about "src/quo2/components/common-styles/unread-grey-dot-style.cljs" or something like this

Reply:
alternatively put it in a "src/quo2/components/common/unread-grey-dot folder with a view and style and then the view can just be `[rn/view {:style components-style/unread-grey-dot}]`. Personally I think this is better 👍

Reply:
Hm, for me, tbh, it seems more convenient to have a common style file exactly for the sake of collecting different unrelated styles that can be placed on the top level. In fact any of our `style` files is a collection of quite unrelated styles. But I don't mind grouping as you suggested either)

Reply:
Hmm, yeah I personally would be strongly against a file of random styles. It will just become a dumping group for random styles that have nothing to do with each other. Just my opinion ofc